directus-extension-charging-stations-bundle/src/endpoints-charging-stations/index.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { defineEndpoint } from '@directus/extensions-sdk';
+
+export default defineEndpoint({
+    id: 'charging-stations',
+    handler: (router, { database }) => {
+        router.post('/update-station-status', async (req, res) => {
```

Comment:
This is a good consideration for when we may have many endpoints in the future. Note, `endpoints-charging-stations` is not in the path because I declared an id for this endpoint. The path is `/charging-stations/update-station-status`, constructed from the id and the url in the router call.